bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/InsteonBinding.java

```diff
@@ -106,15 +106,14 @@
  * @author Rob Nielsen - Port to openHAB 2 insteon binding
  */
 @NonNullByDefault
-@SuppressWarnings({ "null", "unused" })
 public class InsteonBinding {
     private static final int DEAD_DEVICE_COUNT = 10;
 
     private final Logger logger = LoggerFactory.getLogger(InsteonBinding.class);
 
     private Driver driver;
-    private ConcurrentHashMap<InsteonAddress, InsteonDevice> devices = new ConcurrentHashMap<>();
-    private ConcurrentHashMap<String, InsteonChannelConfiguration> bindingConfigs = new ConcurrentHashMap<>();
+    private Map<InsteonAddress, @Nullable InsteonDevice> devices = new ConcurrentHashMap<>();
+    private Map<String, @Nullable InsteonChannelConfiguration> bindingConfigs = new ConcurrentHashMap<>();
```

Comment:
Now that we have EEAs there is no need for annotating the value type with `@Nullable` to prevent those false positives. You should only do that if you put actual `null` values in a map. Which usually is not the case and is also not preferable because it increases the risk of NPEs.

Reply:
fixed